requirements.txt

```diff
@@ -1,20 +1,61 @@
+amqp==5.1.1
+anyio==3.6.2
+asgiref==3.6.0
+async-timeout==4.0.2
+billiard==3.6.4.0
 boto3==1.16.35
 botocore==1.19.35
+build==0.10.0
+cached-property==1.5.2
+celery==5.2.7
+celery-redbeat==2.0.0
+certifi==2022.12.7
+charset-normalizer==3.0.1
+click==8.1.3
+click-didyoumean==0.3.0
+click-plugins==1.1.1
+click-repl==0.2.0
+cycler==0.11.0
 Django==3.1.14
+django-cors-headers==3.11.0
+django-otp==1.1.4
 django-storages==1.10.1
 djangorestframework==3.12.2
+idna==3.4
+importlib-metadata==4.13.0
+jmespath==0.10.0
+kiwisolver==1.4.4
+kombu==5.2.4
+matplotlib==3.3.4
+numpy==1.21.6
```

Comment:
why do we need to add all of these packages? Not sure we use numpy, for example, in the API anywhere

Reply:
Good question. I've just added celery and celery-redbeat. I didn't try to review and optimize other requirements. For example, there is a sense of upgrading the versions as well. Because the code just cannot even run on Python 3.11 and requires pretty old 3.7. But I think that it could be an additional task.

Reply:
Thanks. I see celery and celery-redbeat, but another 40 packages have been added to requirements.txt. Is there a reason for adding them all to the requirements, or can we just add celery and celery-redbeat?

Reply:
That is because I regenerated `requirements.txt` with `pip freeze` command. Your old file did not contain the whole list of dependencies. Many libraries have also their own list of requirements that previously weren't in the file.